tests/test_session.py

```diff
@@ -288,209 +267,11 @@ def test_check_ping_timeout(session, mocker):
     assert session._check_ping_timeout(10, 11)
 
 
-@mocketize
```

Comment:
mocket didn't support `recv_into`. Fortunately the integration tests with a tornado server have the same coverage.